CrewChiefV4/Events/SessionEndMessages.cs

```diff
@@ -130,12 +130,18 @@ public void playFinishMessage(SessionType sessionType, int startPosition, int po
                     {
                         audioPlayer.playMessage(new QueuedMessage(sessionEndMessageIdentifier, AbstractEvent.MessageContents(
                             Position.folderStub + position, folderFinishedRace), 0, null));
+                        // if it's a shit finish, maybe launch into a tirade
+                        if (position - startPosition > 10)
```

Comment:
Sure, crew chief  "knows you're shit" :)  I just wonder  if this has to be taking into account grid size.  Although, 10 pos down is shit :D   Also, what about DQ/DNF?